app/src/main/java/org/simple/clinic/security/pin/PinEntryEvent.kt

```diff
@@ -17,3 +17,5 @@ data class PinAuthenticated(val data: Any?) : PinEntryEvent() {
 }
 
 data class PinVerified(val result: PinVerificationMethod.VerificationResult) : PinEntryEvent()
+
+object PinEntryDoneClicked : PinEntryEvent()
```

Comment:
Can we add an analytics event name for this?

Reply:
Currently, we are not verifying or do anything on this click. So I was planning on adding this once we start doing something? 🤔

Reply:
Ah, good point. It looks good then.